benchmark/ACA-NCM_On-Demand/ACA-NCM gRPC Channels Optimization Report.adoc

```diff
@@ -0,0 +1,95 @@
+= ACA-NCM gRPC Channels Optimization Report
+:revnumber: v1.0
+:revdate: 2021 July 27
+:author: Rio Zhu
+:email: zzhu@futurewei.com>
+
+:toc: right
+:imagesdir: images
+
+== Overview
+
+This document presents the results of running the on-demand workflow tests with Alcor-Control-Agent(ACA), and Network-Configuration-Manager(NCM), with Ignite as the cache layer of NCM.
+
+This report focuses on the testcases where 200 ports will be created on the two physical nodes, where ACAs run, with node one having none of the neighbor information of ports on node two, and node two having all neighbor information of ports on node one. After the ports are created, and the corresponding GoalStates are pushed down to both ACAs, ports on node one will ping ports on node two at a concurrent manner, which triggers 200 concurrent on-demand requests, which are sent by ACA's gRPC client. 
```

Comment:
@zzxgzgz Can you draw one diagram that lists the gRPC client/server on NCM and gRPC client/server on ACA, and details sync/async model, stub and channels, streaming channels and keepalive ping between NCM and ACA?

Reply:
Added.